apps/tomcat/9.0.97/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nexus:
-    image: bitnami/tomcat:9.0.97
+    image: bitnami/tomcat:9.0.98
     restart: always
     container_name: ${CONTAINER_NAME}
     environment:
```

Comment:
This is an incorrect code difference report generated based on outdated documentation from 2021 which includes images and versions that don't necessarily still apply to the current year. In particular, it refers to `tomcat` as if it's version 9.0 instead of version 8.5, with some inconsistencies in how environment variables are specified.

To provide more helpful feedback:
- Check out the latest version of Tomcat image and restart configuration from https://hub.docker.com/r/bitnami/tomcat/
- Ensure your environments variables match what you need (`spring.application.name`, `logging.level`) before proceeding further

Additionally, please update all references in this message with actual current information about Docker image naming conventions, restart settings etc so I can accurately help guide through these discrepancies.